packages/wonder-blocks-button/components/button.vite_test.js

```diff
@@ -0,0 +1,19 @@
+/* global render */
+// @flow
+import * as React from "react";
+
+import Button from "./button.js";
+
+describe("foo", () => {
+    test("it should render a button", async () => {
+        // Arrange
+        // $FlowFixMe
```

Comment:
Why is this FlowFixMe?

Reply:
I'll add a note.

Reply:
The problem is that flow can resolve this global.  I hope that we can make it work the same way other jest globals do.